pkgdb2/lib/__init__.py

```diff
@@ -2038,75 +2038,75 @@ def vcs_acls(session, eol=False, oformat='text', skip_pp=None):
     output = {}
     pkgs = model.vcs_acls(session=session, eol=eol)
     if oformat == 'json':
-        for pkg in pkgs:
+        for pkgname, username, branchname in pkgs:
             user = None
             group = None
-            if pkg[1] and pkg[1].startswith('group::'):
-                    group = pkg[1].replace('group::', '')
+            if username and username.startswith('group::'):
+                    group = username.replace('group::', '')
             else:
-                user = pkg[1]
+                user = username
 
-            if pkg[0] not in output:
-                output[pkg[0].encode('utf-8')] = {}
+            if pkgname not in output:
+                output[pkgname.encode('utf-8')] = {}
```

Comment:
I think that's a left over from when I tried to build the JSON by hand using a dict structure.